dashboard/lib/pd/certificate_renderer.rb

```diff
@@ -31,11 +31,9 @@ def self.render_workshop_certificate(enrollment)
       [
         {
           string: enrollment.try(:full_name) || '',
-          pointsize: 90,
-          height: 100,
-          width: 1200,
```

Comment:
For my own education: why are we getting rid of the height and width here?

Reply:
With the annotations API before, you could set the height and width in order to limit the bounds of of the text area. Mostly useful for making sure your text isn't too wide or to tall. That could be useful so we protect ourselves against user who troll us and create names which are very long. I will play around with it before I submit this.